estela-api/core/models.py

```diff
@@ -411,3 +411,28 @@ class UsageRecord(models.Model):
 
     class Meta:
         ordering = ["-created_at"]
+
+
+class Activity(models.Model):
+    aid = models.AutoField(
+        primary_key=True, help_text="A unique integer value identifying this activity."
+    )
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="activities",
+        help_text="User who performed the activity.",
+    )
+    project = models.ForeignKey(
+        Project,
+        on_delete=models.CASCADE,
+        related_name="activities",
+        help_text="Project to which the activity corresponds.",
+    )
+    created = models.DateTimeField(
+        auto_now_add=True, editable=False, help_text="Activity creation date."
```

Comment:
Let's use `Date when the activity was performed.`. It's more descriptive of what it refers to.